assets/js/wc-gateway-ppec-generate-cart.js

```diff
@@ -30,52 +30,66 @@
 			$( '#woo_pp_ec_button_product > *' ).css( 'pointer-events', 'none' );
 		} );
 
+	// True if the product is simple or the user selected a valid variation. False on variable product without a valid variation selected
+	var variation_valid = true;
+
+	// True if all the fields of the product form are valid. False otherwise
+	var fields_valid = true;
+
+	// True if there's an automatic form validation being performed (not user-initiated), so the user must not be alerted. False otherwise
+	var silent_validation = false;
```

Comment:
Since we're using `checkValidity()` and it doesn't show show validation errors to the user, we can remove `silent_validation` from all the parts of the code, right?

Reply:
Same for lines 47 & 49 below.

Reply:
🤦‍♂️ Yes. Done